app/src/componentLibrary/attributeList/attributeList.jsx

```diff
@@ -38,6 +39,7 @@ export const AttributeList = ({
   customClass,
   showButton,
   editable,
+  attributesListClassname,
```

Comment:
As it is a `AttributeList` component, we expect that `className` or `customClass` will be applied for the component's top level element.
On the other hand custom classes for some inner stuff may have specific names, f.e. `attributeClassName` or smth like that.